### PR TITLE
add missing pieces in exercise section

### PR DESCRIPTION
**Pull request by henderiw**
some missing pieces added in step 003 of the exercise

### CODE REVIEW

File: user-guide/exercises.md [hunk 1]
```diff
@@ -397,12 +397,19 @@ packagevariant.config.porch.kpt.dev/network created
...
 topology. This will likely be automated in the future, but for now you can
 just directly apply the configuration we have created that matches this test
-topology.
+topology. Within this step also the credentials and information is provided
+to configure the network device, that aligns with the topology.
 
 ```bash
+kubectl apply -f test-infra/e2e/tests/003-secret.yaml
+
+export LEAF_IP=$(docker inspect net-free5gc-net-leaf -f '{{.NetworkSettings.Networks.kind.IPAddress}}')
```
[the remark this answers is not in the document]
Comment (henderiw) [reply user-guide/exercises.md:409]: ubuntu@nephio-r1-e2e:~$ ./test-infra/e2e/provision/hacks/network-topo.sh
./test-infra/e2e/provision/hacks/network-topo.sh: line 18: TESTDIR: unbound variable

[the remark this answers is not in the document]
Comment (johnbelamaric) [reply user-guide/exercises.md:409]: Doh

[the remark this answers is not in the document]
Comment (johnbelamaric) [reply user-guide/exercises.md:409]: https://github.com/nephio-project/test-infra/pull/148

**Comment by matysiaq** [comment]
As I understand, commands in the instruction are provided to be performed outside test-infra. So this line:

`./e2e/provision/hacks/network-topo.sh`

suppose to be 

`./test-infra/e2e/provision/hacks/network-topo.sh`

File: user-guide/exercises.md [hunk 2]
```diff
@@ -397,12 +397,15 @@ packagevariant.config.porch.kpt.dev/network created
...
-However, we will also need to configure the resource backend for our particular
+However, we also need to configure the resource backend for our particular
 topology. This will likely be automated in the future, but for now you can
 just directly apply the configuration we have created that matches this test
-topology.
+topology. Within this step also the credentials and information is provided
+to configure the network device, that aligns with the topology.
 
 ```bash
+./e2e/provision/hacks/network-topo.sh
```
Comment (johnbelamaric) [inline user-guide/exercises.md:407]:
```suggestion
./test-infra/e2e/provision/hacks/network-topo.sh
```

**Pull request merged**